test/unit/matrix/test_retiled_matrix.cpp

```diff
@@ -78,7 +78,7 @@ const std::vector<std::tuple<LocalElementSize, TileElementSize, LocalTileSize>>
     {{4, 4}, {4, 4}, {1, 1}},
 });
 
-TYPED_TEST(RetiledMatrixTest, LocalConstructor) {
+TYPED_TEST(RetiledMatrixLocalTest, LocalConstructor) {
```

Comment:
Why was this renamed? Am I missing something obvious? :thinking:

Reply:
The dependencies test should also be changed to a local one. Happy to move this to a separate PR if needed (they are essentially un-related with this PR).

Reply:
Ah, thanks! Yes, that makes sense now. If you don't mind, a separate PR would be great along with this explanation.

Reply:
Moved to https://github.com/eth-cscs/DLA-Future/pull/1216. I also moved there the small documentation fixes, to de-clutter a bit more this PR.